app/javascript/components/partnering_organizations/PartneringOrganizationNavbar.js

```diff
@@ -124,6 +124,7 @@ class PartneringOrganizationNavbar extends React.Component {
           <div className="pt-navbar-group pt-align-right">
             <a href={`/partnering_organizations/resources`}><button className="pt-button pt-minimal pt-icon-document">Resources</button></a>
             <span className="pt-navbar-divider"></span>
+            <a href={`/partnering_organizations/${this.props.partneringOrganization.id}/subscriptions`}><button className="pt-button pt-minimal pt-icon-document">Subscribers</button></a>
```

Comment:
I think you can actually delete `/${this.props.partneringOrganization.id}` since we're just passing in the prop!